app/views/authors/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2>Edit an Author</h2>
+<%= form_tag author_path, method: "patch" do %>
+  <% if @author.errors.any? %>
+    <div id="error_explanation">
+      <h2>There were some errors:</h2>
+      <ul>
+        <% @author.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field <%= 'field_with_errors' if @author.errors[:name].any? %>">
+    <%= label_tag "name", "Name" %>
+    <%= text_field_tag "name", @author.name %>
+  </div>
+
+  <div class="field <%= 'field_with_errors' if @author.errors[:email].any? %>">
+    <%= label_tag "email", "Email" %>
+    <%= text_field_tag "email", @author.email %>
+  </div>
+
+  <div class="field <%= 'field_with_errors' if @author.errors[:phone_number].any? %>">
+    <%= label_tag "phone_number", "Phone Number" %>
+    <%= text_field_tag "phone_number", @author.phone_number %>
+  </div>
+
+  <%= submit_tag "Update" %>
+<% end %>
```

Comment:
We talked about it yesterday, but would be nice to see `form_for` going forward.